src/gui/widgets/mapper_proxystyle.cpp

```diff
@@ -95,6 +95,16 @@ void MapperProxyStyle::drawPrimitive(QStyle::PrimitiveElement element, const QSt
 			drawSegmentedButton(segment, overridden_element, option, painter, widget);
 			return;
 		}
+		if (touch_mode && element == PE_PanelButtonTool)
+		{
+			auto const& palette = option->palette;
+			auto const& window = palette.window().color();
```

Comment:
Suggest calling the variable `window_color` or `base_color` instead of plain `window`.